sky/provision/kubernetes/network.py

```diff
@@ -0,0 +1,79 @@
+import os
+from typing import Any, Dict, List, Optional, Tuple
+
+from sky.provision.kubernetes import network_utils
+
+PATH_PREFIX = "/skypilot/{cluster_name_on_cloud}/{port}"
```

Comment:
```suggestion
_PATH_PREFIX = "/skypilot/{cluster_name_on_cloud}/{port}"
```
nit: start with underscore if it is an inner constant

Reply:
Done in https://github.com/skypilot-org/skypilot/pull/2713/commits/d3fc952568a4ddf853355b15006b42fecd737e32